projects/landing-page/webpack.config.js

```diff
@@ -0,0 +1,78 @@
+/* eslint-disable no-undef */
```

Comment:
Sure. This can be done in different PR though. The purpose of this pr is to remove parrcel, and not create common configs

Reply:
ok :+1: but let's do this quickly after this merges

Reply:
> to move common webpack configuration into a file in the root 

But why?
There's nothing wrong with configuring multiple projects differently. If projects can in theory have different configurations, then the configuration files should be separate.
It's not because files are similar that you should de-duplicate them. You should only de-duplicate them if it makes sense for the multiple different projects to always conform to the same configuration, which isn't the case.

Reply:
It is the case. The vast majority of the webpack configuration is identical across landing page, burnr and the demo.  It's true that they can in theory be different but in practice they're not.  It's not easy at the moment to see what's specific about the configuration for each project and if you make changes to the webpack configuration you have to remember to do it in all the places for them to stay consistent.  The configuration for:
* babel
* css loading
* HTML plugin
* typsecript loading

is always going to be the same across these projects.  If we change them we're going to want to change it everywhere.  Having it one place makes that less laborious

Reply:
IMO webpack configurations should be dedicated per projects. Finding common parts of configurations an making a common one that then will be inherited to the rest, that will add the extra configs makes it more spaghetti. Common parts of webpack configs are very little and in some cases even these are different. A good example is the `ts-loader` where in `demo` we use it without extra options while in `burnr` we add options like `transpileOnly` and `projectReferences`. I would say that its better to narrow down the webpack confg per project and remove unnecessary parameters that probably exists due to copy/pasting and if there is some obvious added value then to create a common webpack file.

Reply:
> if you make changes to the webpack configuration you have to remember to do it in all the places for them to stay consistent.

If you merge all webpack configurations into one, then if you make changes to the webpack configuration you have to make sure that all projects still work.

Reply:
I'm not advocating merging it all into one .. just pulling the common boilerplate out

Reply:
Added an [issue](https://github.com/paritytech/substrate-connect/issues/997)